### PR TITLE
Majority Element of an Array

### LINKED ISSUE #1367: Find the majority element of an array in C, C++, Java, Python and Go
@marwahmanbir @rishabhgarg25699 Please assign it to me.

### PR DESCRIPTION
**Pull request by singh-shreya6**
Fixes #1367 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added implementation for finding majority element of an array

#### Languages Used:
- C
- C++
- Java
- Python
- Go

#### Files Added:
- Majority_Element.c
- Majority_Element.cpp
- Majority_Element.java
- Majority_Element.py
- Majority_Element.go

### CODE REVIEW

File: Majority_Element/Majority_Element.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+int checkMajorityElement(int arr[], int N)
+{
+    int freq[SIZE] = {0};
+    // Computing frequency of each element
+    for (int i = 0; i < N; i++)
+        freq[arr[i]]++;
+    for (int i = 0; i < SIZE; i++)
+    {
+        if(freq[i] != 0 && freq[i] > (N / 2))
```
Comment (MastersAbh) [inline Majority_Element/Majority_Element.c]: space after if

File: Majority_Element/Majority_Element.cpp [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+int checkMajorityElement(int arr[], int N)
+{
+    map<int, int> mp;
+    // Computing frequency of each element using Map
+    for (int i = 0; i < N; i++)
+        mp[arr[i]]++;
+    map<int, int> :: iterator it;
+    for (it = mp.begin(); it != mp.end(); it++)
+    {
+        if(it -> second > (N / 2))
```
Comment (MastersAbh) [inline Majority_Element/Majority_Element.cpp]: space after if

File: Majority_Element/Majority_Element.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        for (int i = 0; i < N; i++) 
+        { 
+            if (mp.containsKey(arr[i]))
+                mp.put(arr[i], mp.get(arr[i]) + 1); 
+            else 
+                mp.put(arr[i], 1); 
+        } 
+        for (Map.Entry<Integer, Integer> entry : mp.entrySet())
+        {
+            if(entry.getValue() > (N / 2))
```
Comment (MastersAbh) [inline Majority_Element/Majority_Element.java]: space after if

**Comment by singh-shreya6** [comment]
Done @MastersAbh. Please review.

**Pull request merged**